packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.class.js

```diff
@@ -5,6 +5,24 @@ import { BACKUP_STATE_DISABLED, BACKUP_OFFER_LEGACY } from './backup.constants';
 export default class Backup {
   constructor(backup) {
     Object.assign(this, backup);
+    this.initializeBackup();
+  }
+
+  initializeBackup() {
+    this.backupSizeInReport =
+      this.backupSizeInReport === null ? false : this.backupSizeInReport;
```

Comment:
`this.backupSizeInReport ? false : this.backupSizeInReport;` should work for `null` check